### PR TITLE
Basic design

### PR DESCRIPTION
**Pull request by Kerub88**

### CODE REVIEW

File: src/js/login.js [hunk 1]
```diff
@@ -7,20 +7,36 @@ class Login extends Component {
...
+            status: 'hidden',
+            loginOrRegis: 'login',
             errorMessage: 'error: '
         };
         let ajax = new Ajax();
         ajax.get('/tokenverify', this.tokenGetter.bind(this), localStorage.getItem("token"));
     }
+
+    StateCheck() {
+        if (this.state.status === 'hidden') {
```
Comment (tiborsaas) [inline src/js/login.js]: use a more sane code, like negating a boolean value

File: src/js/login.js [hunk 2]
```diff
@@ -7,20 +7,36 @@ class Login extends Component {
...
+            this.setState({status: 'not_hidden'});
+        } else {
+            this.setState({status: 'hidden'});
+        }
+        if (this.state.loginOrRegis === 'login') {
```
Comment (tiborsaas) [inline src/js/login.js]: loginOrRegis is not a good name, call it more generic like action or displayedForm

File: src/js/login.js [hunk 3]
```diff
@@ -38,41 +54,45 @@ class Login extends Component {
...
-    }
-    RegisterOnClick() {
-        let ajax = new Ajax();
-        let res = JSON.stringify({
-            username: this.state.inputLoginValue,
-            password: this.state.inputPassValue
-        });
-        let url = '/registerdbcheck';
-        ajax.post(url, this.dataHandler.bind(this), res);
+    ConfirmOnClick() {
```
Comment (tiborsaas) [inline src/js/login.js]: don't capitalize method names

File: src/css/login.scss [hunk 4]
```diff
@@ -80,28 +77,53 @@ h2 {
     color: white;
 }
 
-.login_name::before {
-    transition: transform .35s ease-out 50ms,width .35s ease-out 50ms,height 0ms ease-out .4s;
+// .experiment {
```
Comment (tiborsaas) [inline src/css/login.scss]: remove commented out styles

**Comment by Kerub88** [comment]
refact(login): refactored but it broke everything. Its not shorter or more readable. ternary version is commented out

**Pull request merged**